HiP-DataStore/appsettings.json

```diff
@@ -1,8 +1,45 @@
 ﻿{
+  "Port": 5000,
   "Logging": {
     "IncludeScopes": false,
     "LogLevel": {
       "Default": "Warning"
     }
+  },
+
+  //CORS settings for Multiple environments the application is currently running in [Development, Staging, Production]
```

Comment:
I think, strictly speaking, JSON doesn't support comments. But the JSON parser here seems to be fine with comments anyways.